src/anchor/idl.json

```diff
@@ -640,6 +640,42 @@
           }
         }
       ]
+    },
+    {
+      "name": "initPriceFeedIndex",
+      "discriminant": { "value": [2, 0, 0, 0, 19, 0, 0, 0] },
+      "accounts": [
+        {
+          "name": "fundingAccount",
+          "isMut": true,
+          "isSigner": true
+        },
+        {
+          "name": "priceAccount",
+          "isMut": true,
+          "isSigner": false
+        },
+        {
+          "name": "permissionsAccount",
+          "isMut": true,
```

Comment:
We also need to change `permissionsAccount` to be writable in the declaration of `addPrice`.

Reply:
Can you also update the docs for those methods in your PR? it seems they are deprecated.

Reply:
Updated.